zerver/lib/export.py

```diff
@@ -2038,21 +2038,42 @@ def get_single_user_config() -> Config:
     return user_profile_config
 
 
+def chunkify(lst: List[int], chunk_size: int) -> List[List[int]]:
+    # chunkify([1,2,3,4,5], 2) == [[1,2], [3,4], [5]]
+    result = []
+    i = 0
+    while True:
+        chunk = lst[i : i + chunk_size]
+        if len(chunk) == 0:
+            break
+        else:
+            result.append(chunk)
+            i += chunk_size
+
+    return result
```

Comment:
We could make this fancier with `yield`, e.g.

https://www.geeksforgeeks.org/break-list-chunks-size-n-python/

Not sure it's worth it.

Reply:
`do_batch_update` is one slicker version of this for database migrations, but it's not directly usable here since it does the queries and everything.

Reply:
I just left it as is.